pkg/chaoskernel/pb/bpfki.pb.go

```diff
@@ -393,7 +393,7 @@ func init() {
 	proto.RegisterType((*StatusResponse)(nil), "bpfki.StatusResponse")
 }
 
-func init() { proto.RegisterFile("bpfki.proto", fileDescriptor_62eed357eb71de0e) }
+func init() { protoregistry.GlobalFiles.RegisterFile("bpfki.proto", fileDescriptor_62eed357eb71de0e) }
```

Comment:
Code generated by protoc-gen-go.  **DO NOT EDIT.**. The correct way to fix `deprecated` in this file may be updating the protoc / protobuf / grpc and regenerate this file. However, it could be a little complicated.